index.js

```diff
@@ -33,7 +33,8 @@ module.exports = function(app) {
       udpSocket = require('dgram').createSocket('udp4')
       unsubscribe = Bacon.combineWith(function(position, sog, cog, head) {
         return createPositionReportMessage(mmsi, position.latitude, position.longitude, mpsToKn(sog), cog, head)
-      }, ['navigation.position', 'navigation.speedOverGround', 'navigation.courseOverGroundTrue', 'navigation.headingTrue'].map(app.streambundle.getSelfStream, app.streambundle)).changes().debounceImmediate(60000).onValue(msg => {
+      }, ['navigation.position', 'navigation.speedOverGround', 'navigation.courseOverGroundTrue', 'navigation.headingTrue'].map(app.streambundle.getSelfStream, app.streambundle)).changes().debounceImmediate(props.updaterate*1000).onValue(msg => {
```

Comment:
Please add a default for updaterate, as any pre existing config files will otherwise fail.
```(props.updaterate || 60)*1000``` should do it.